stream-chat-android-client/build.gradle

```diff
@@ -19,12 +19,11 @@ ext {
 apply from: "${rootProject.projectDir}/scripts/publish-mavencentral.gradle"
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion Configuration.compileSdkVersion
     buildToolsVersion "29.0.3"
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 28
-        versionCode 1
+        minSdkVersion Configuration.minSdkVersion
+        targetSdkVersion Configuration.targetSdkVersion
```

Comment:
These values were previously set directly both here and in the `offline` module. As a result of this, the target and compile SDKs were already lagging behind the other modules.